wes_client/client.py

```diff
@@ -0,0 +1,121 @@
+from logging import Logger
+import socket
+
+import requests
+from requests import Session
+import urllib3
```

Comment:
As you need only the exception, I'd do just `from urllib3 import NewConnectionError`